### ISSUE TITLE
 UNUSED_METHOD_ARGUMENT is triggered with a function argument

### ISSUE THREAD

**Issue by JuanTorr** [cflint/CFLint#550]
For example:

```
public string function myFunction(function doSomething){
    return doSomething();
}
```
`UNUSED_METHOD_ARGUMENT: Argument dosomething is not used in function. Consider removing it.`

**Comment by KamasamaK** [comment]
I have confirmed that the example given above incorrectly reports the UNUSED_METHOD_ARGUMENT issue.

However, I have found that it does not report that issue when including the scope as below:
```
public string function myFunction(function doSomething){
    return arguments.doSomething();
}
```
